invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dublin Core based Schema for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing
+
+
+class DublinCore(Schema):
```

Comment:
In general we suffix the name with `Schema` to be more clear in other places when importing the class :)
```suggestion
class DublinCoreSchema(Schema):
```